client/src/legacy/GridField.js

```diff
@@ -9,6 +9,12 @@ require('../../../thirdparty/jquery-entwine/dist/jquery.entwine-dist.js');
 
 $.entwine('ss', function($) {
   $('.grid-field').entwine({
+    onmatch: function () {
+      if (this.needsColumnFix()) {
+        this.fixColumns();
+        this.injectSearchButton(false);
```

Comment:
for consistency, perhaps both can include the param

Reply:
Whoops. Yeah, I had done it that way initially, but it looks like the `fixColumns` function is no longer using the `visible` param, so it should just be removed from the definition.